Runtimes/Supplemental/cmake/modules/FindSwiftPlatformOverlay.cmake

```diff
@@ -0,0 +1,269 @@
+      list(APPEND swiftGlibc_LIBRARY_HINTS
+        "${Swift_SDKROOT}/usr/lib/swift_static/linux")
+      list(APPEND swiftGlibc_NAMES libswiftGlibc.a)
+    else()
+      list(APPEND swiftGlibc_INCLUDE_DIR_HINTS
+        "${Swift_SDKROOT}/usr/lib/swift/linux/")
+      list(APPEND swiftGlibc_LIBRARY_HINTS
+        "${Swift_SDKROOT}/usr/lib/swift/linux")
+      list(APPEND swiftGlibc_NAMES libswiftGlibc.so)
+    endif()
+    set(swiftGlibc_MODULE_NAME "Glibc.swiftmodule")
+  endif()
+elseif(WIN32)
+  list(APPEND OVERLAY_TARGET_NAMES "swiftWinSDK")
+  list(APPEND OVERLAY_TARGET_NAMES "swiftCRT")
+
+  # If the swiftWinSDK_DIR_FLAG is specified, look there instead. The cmake-generated
+  # config file is more accurate
+  if(swiftWinSDK_DIR)
+    if(swiftWinSDK_FIND_REQUIRED)
+      list(APPEND args REQUIRED)
+    endif()
+    if(swiftWinSDK_FIND_QUIETLY)
+      list(APPEND args QUIET)
+    endif()
+    find_package(swiftWinSDK NO_MODULE ${args})
+  # Look in the SDK
+  else()
+    list(APPEND swiftWinSDK_INCLUDE_DIR_HINTS
+      "${Swift_SDKROOT}/usr/lib/swift/windows"
+      "$ENV{SDKROOT}/usr/lib/swift/windows")
+    list(APPEND swiftWinSDK_LIBRARY_HINTS
+      "${Swift_SDKROOT}/usr/lib/swift/${${PROJECT_NAME}_PLATFORM_SUBDIR}/${${PROJECT_NAME}_ARCH_SUBDIR}"
+      "${Swift_SDKROOT}/usr/lib/swift"
+      "$ENV{SDKROOT}/usr/lib/swift/${${PROJECT_NAME}_PLATFORM_SUBDIR}/${${PROJECT_NAME}_ARCH_SUBDIR}"
+      "$ENV{SDKROOT}/usr/lib/swift")
+    list(APPEND swiftWinSDK_NAMES swiftWinSDK.lib)
+    list(APPEND swiftWinSDK_MODULE_NAME "WinSDK.swiftmodule")
+  endif()
+
+  # If the swiftCRT_DIR_FLAG is specified, look there instead. The cmake-generated
+  # config file is more accurate
+  if(swiftCRT_DIR)
+    if(swiftCRT_FIND_REQUIRED)
+      list(APPEND args REQUIRED)
+    endif()
+    if(swiftCRT_FIND_QUIETLY)
+      list(APPEND args QUIET)
+    endif()
+    find_package(swiftCRT NO_MODULE ${args})
+  # Look in the SDK
+  else()
+    list(APPEND swiftCRT_INCLUDE_DIR_HINTS
+      "${Swift_SDKROOT}/usr/lib/swift/windows"
+      "$ENV{SDKROOT}/usr/lib/swift/windows")
+    list(APPEND swiftCRT_LIBRARY_HINTS
+      "${Swift_SDKROOT}/usr/lib/swift/${${PROJECT_NAME}_PLATFORM_SUBDIR}/${${PROJECT_NAME}_ARCH_SUBDIR}"
+      "${Swift_SDKROOT}/usr/lib/swift"
+      "$ENV{SDKROOT}/usr/lib/swift/${${PROJECT_NAME}_PLATFORM_SUBDIR}/${${PROJECT_NAME}_ARCH_SUBDIR}"
+      "$ENV{SDKROOT}/usr/lib/swift")
+    list(APPEND swiftCRT_NAMES swiftCRT.lib)
+    list(APPEND swiftCRT_MODULE_NAME "CRT.swiftmodule")
+  endif()
+elseif(ANDROID)
+  list(APPEND OVERLAY_TARGET_NAMES "swiftAndroid")
+
+  # If the swiftAndroid_DIR_FLAG is specified, look there instead. The cmake-generated
+  # config file is more accurate
+  if(swiftAndroid_DIR)
+    if(swiftAndroid_FIND_REQUIRED)
+      list(APPEND args REQUIRED)
+    endif()
+    if(swiftAndroid_FIND_QUIETLY)
+      list(APPEND args QUIET)
+    endif()
+    find_package(swiftAndroid NO_MODULE ${args})
```

Comment:
This is part of the platform overlay package. This should be exported by the SwiftPlatformOverlay module.